.github/workflows/ci_release.yml

```diff
@@ -26,16 +26,11 @@ jobs:
     name: "Checks and Tests"
     runs-on: ubuntu-latest
 
-    strategy:
-      fail-fast: false
-      matrix:
-        node-version: [14.x, 16.x]
-
     steps:
       - name: "Run Yarn Basic Checks"
         uses: SkynetLabs/.github/.github/actions/yarn-basic-checks@master
         with:
-          node-version: ${{ matrix.node-version }}
+          node-version: 18.x
```

Comment:
Looks like there is some issue with Node 18 (checks are failing). We should do `16` to be consistent with our `Dockerfile`

Reply:
updated to 16.x in https://github.com/SkynetLabs/webportal-accounts-dashboard/pull/56/commits/9ecdbf6baab85249356f06de5ee17f8c484168fc